app/models/qernel/graph_api/energy.rb

```diff
@@ -93,18 +95,32 @@ def group_of_excess_events(durations = [], excludes = [])
 
       # Public: Returns total number of hours there was excess
       #
-      # Returns an Integer
+      # Returns an Integer (hours)
       def total_number_of_excess_events(excludes = [])
         graph.plugin(:merit).order.excess(excludes).total_number_of_events
       end
 
       # Public: Returns number of blackout hours
       #
-      # Returns an Integer
-      def number_of_blackout_hours
+      # Returns an Integer (hours)
+      def number_of_power_shortage_hours
         graph.plugin(:merit).order.blackout.number_of_hours
       end
 
+      # Public: Returns the peak of blackout hours, defined as the largest single hour deficit
+      #
+      # Returns an Integer (MW)
```

Comment:
@louispt1 I might miss some developer terminology, but integer means it's a whole number right? This returns a number with decimals, shouldn't be a float?